R_code/fingerprintplots.r

```diff
@@ -0,0 +1,202 @@
+# plotting GPP vs. R and color coding by season; JAZ & JK; 2018-03-09
+
+#first part is creating dataframes with data we want to plot.
+#starts with reading in individual lake files and binding them into one data frame
+#then merges dataframes together (e.g., metabolism dataframe with loading data frame)
+#there is code in the dropbox for creating a dataframe for the light climate data but I've left this
+#out for now because the light climate data are not on Github
+#code for plotting starts~line 100
+
+
+library(xlsx)
+
+dir<-'C:/Users/jklug/Documents/catchment_metab_wg/results/metab/20161107/' # directory of metabolism data
```

Comment:
If you are using projects in R Studio, you should need your full directory path, just the `results/metab/20161107`. This allows for others to use this code without having to change the directory path. 
```suggestion
dir<-'results/metab/20161107/' # directory of metabolism data
```